engine/src/test/scala/cromwell/engine/io/nio/NioFlowSpec.scala

```diff
@@ -175,53 +174,53 @@ class NioFlowSpec extends TestKitSuite with AsyncFlatSpecLike with Matchers with
     }
   }
 
-  it should "fail if DrsPath hash doesn't match checksum" in {
-    val testPath = mock[DrsPath]
-    when(testPath.limitFileContent(any[Option[Int]], any[Boolean])(any[ExecutionContext])).thenReturn("hello".getBytes)
-    when(testPath.getFileHash).thenReturn(
-      FileHash(HashType.Crc32c, "boom")
-    ) // correct Base64-encoded crc32c checksum is "9a71bb4c"
-
-    val context =
-      DefaultCommandContext(contentAsStringCommand(testPath, Option(100), failOnOverflow = true).get, replyTo)
-    val testSource = Source.single(context)
-
-    val stream = testSource.via(flow).toMat(readSink)(Keep.right)
-
-    stream.run() map {
-      case (IoFailure(_, EnhancedCromwellIoException(_, receivedException)), _) =>
-        receivedException.getMessage should include("Failed checksum")
-      case (ack, _) => fail(s"read returned an unexpected message:\n$ack\n\n")
-    }
-  }
-
-  it should "retry if DrsPath hash check fails, then succeed if the second check passes" in {
-    val testPath = mock[DrsPath]
-    when(testPath.limitFileContent(any[Option[Int]], any[Boolean])(any[ExecutionContext]))
-      .thenReturn("hola".getBytes)
-      .thenReturn("hello".getBytes)
-      .thenReturn("hello".getBytes)
-    when(testPath.getFileHash)
-      .thenReturn(FileHash(HashType.Crc32c, "9a71bb4c"))
-      .thenReturn(FileHash(HashType.Crc32c, "boom"))
-      .thenReturn(FileHash(HashType.Crc32c, "9a71bb4c"))
-
-    val context =
-      DefaultCommandContext(contentAsStringCommand(testPath, Option(100), failOnOverflow = true).get, replyTo)
-    val testSource = Source.single(context)
-
-    val stream = testSource.via(flow).toMat(readSink)(Keep.right)
-
-    stream.run() map { result =>
-      verify(testPath, times(3)).limitFileContent(any[Option[Int]], any[Boolean])(any[ExecutionContext])
-      verify(testPath, times(3)).getFileHash
-      result match {
-        case (success: IoSuccess[_], _) => assert(success.result.asInstanceOf[String] == "hello")
-        case (ack, _) =>
-          fail(s"read returned an unexpected message:\n$ack\n\n")
-      }
-    }
-  }
+//  it should "fail if DrsPath hash doesn't match checksum" in {
```

Comment:
Should this be removed?

Reply:
Oh man good catch, I commented out these tests to "fix later" and then forgot about them. Later is now, will fix.

Reply:
I resolved this by removing the functionality checked by these tests.